src/hugr/views/hierarchy.rs

```diff
@@ -54,6 +53,20 @@ where
     _phantom: std::marker::PhantomData<Root>,
 }
 
+impl<'g, Root, Base> SiblingGraph<'g, Root, Base>
+where
+    Base: HugrInternals,
+{
+    /// Helper function to initialize the internal portgraph view.
+    fn init_graph(hugr: &Base, root: Node) -> FlatRegionGraph<'_> {
```

Comment:
AFAICS these two are used exactly once - in quite small calling methods - if the goal is to reuse this elsewhere (in a later PR), might be better to do this pulling out in that PR?

Reply:
They where used in the rolled-back changes (that will come in a following PR once portgraph is updated).
I'll clean this up.